File: EnviroScope_ServerSide/CollectorAPI/dependecies/db_config.js
const { mysql } = require('./modules');

const db = mysql.createConnection({
    host: '127.0.0.2',
    port: 3306,  
    user: 'root',
    password: '$_Jon_Xhaxhi_123_@_$',
    database: 'enviro_scope',
});

module.exports = db;

File: EnviroScope_ServerSide/PredictionAPI/db_config.py
import mysql.connector

def connect_to_database():
    try:
        connection = mysql.connector.connect(
            
            host= '127.0.0.2',
            user= 'root',
            password= '$_Jon_Xhaxhi_123_@_$',
            database= 'enviro_scope',
            
        )

        if connection.is_connected():
            print('Connected to MySQL database')
            return connection
    except Exception as e:
        print('Error connecting to database:', e)
        return None
